### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/kimai/messages/) for Kimai - Time tracking/Main application.

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by kevinpapst** [comment]
Thanks @BeckeBauer ! Can you let me know here when you are "done". Then I keep this PR open until it is finished

**Comment by BeckeBauer** [comment]
I will... my colleagues still need a bit...

**Comment by BeckeBauer** [comment]
okay, ready!

File: translations/export.ko.xlf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="ko" datatype="plaintext" original="export.en.xlf">
+    <body>
+      <trans-unit id="default.pdf.twig">
+        <source>default.pdf.twig</source>
+        <target></target>
```
Comment (kevinpapst) [inline translations/export.ko.xlf:7]: empty translations

File: translations/email.ko.xlf [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="ko" datatype="plaintext" original="email.en.xlf">
+    <body>
+      <trans-unit id="button_copy_url_to_browser">
+        <source>button_copy_url_to_browser</source>
+        <target></target>
```
Comment (kevinpapst) [inline translations/email.ko.xlf:7]: empty translations

File: translations/teams.ko.xlf [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="ko" datatype="plaintext" original="teams.en.xlf">
+    <body>
+      <trans-unit id="teams.title">
+        <source>teams.title</source>
+        <target></target>
```
Comment (kevinpapst) [inline translations/teams.ko.xlf:7]: empty translations

File: translations/reporting.ko.xlf [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="ko" datatype="plaintext" original="reporting.en.xlf">
+    <body>
+      <trans-unit id="reporting.title">
+        <source>reporting.title</source>
+        <target></target>
```
Comment (kevinpapst) [inline translations/reporting.ko.xlf:7]: empty translations

**Review by kevinpapst — COMMENTED** [review]
@BeckeBauer 4 of the Korean translation files have no translations. empty translations break kimai. I can either delete the files or do you pan to add these as well?

**Comment by BeckeBauer** [comment]
strange 
Screenshot 2020-11-06 134638
How do we find the missing translations?

**Comment by kevinpapst** [comment]
That was a mistake in Weblate ... or my usage of it. I told Weblate to push the changes again and now they are available in GitHub.

**Review by kevinpapst — APPROVED** [review]
Thanks @BeckeBauer and colleagues 😄

**Pull request merged**